packages/activemq_artemis/data_stream/acceptor/fields/base-fields.yml

```
@@ -0,0 +1,12 @@
+- name: data_stream.type
+  type: constant_keyword
+  description: Data stream type.
```

You can use the `external: ecs` definition for these fields that exist in ECS to ensure there is a consistent definition across integrations. See [Defining an ECS field without using an external definition](https://github.com/elastic/integrations/wiki/Fleet-Package-Code-Review-Comments#defining-an-ecs-field-without-using-an-external-definition).

```suggestion
- name: data_stream.type
  external: ecs
```

Or if you want an automatic fix you can run 

`go run github.com/andrewkroh/fydler@main -fix packages/activemq_artemis/**/fields/*.yml`